test/WebJobs.Script.Tests/Configuration/ScriptJobHostOptionsSetupTests.cs

```diff
@@ -230,50 +196,44 @@ public void Configure_AppliesRetry_ExponentialBackOffDelay(string expectedStrate
             Assert.Equal(maxIntervalTimeSpan, options.Retry.MaximumInterval);
         }
 
-        [Fact]
-        public void Configure_TimeoutDefaultsNull_IfNotDynamic()
```

Comment:
Renamed to Configure_AppliesExpectedFunctionTimeoutDefault_ForNonConsumptionSkus and moved above next to the Consumption version.